src/applications/static-pages/simple-forms/40-0247/App.js

```diff
@@ -0,0 +1,176 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
+import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
+
+const App = ({ formEnabled }) => {
+  if (formEnabled === undefined) {
+    return <va-loading-indicator message="Loading..." />;
+  }
+
+  if (formEnabled) {
+    return (
+      <>
+        <p>You can apply for a PMC in any of these 5 ways.</p>
+        <h3>Option 1: Apply online</h3>
+        <a
+          className="vads-c-action-link--blue"
```

Comment:
Could you make this link green? It should be a primary action link. https://design.va.gov/components/link/action#primary

@tlei123

Reply:
@megzehn , yes; className changed to "vads-c-action-link--grean".